tests/std/tests/P0295R0_gcd_lcm/test.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <limits.h>
+#include <numeric>
+
+using std::gcd;
+using std::lcm;
+
+#define STATIC_ASSERT(...) static_assert(__VA_ARGS__, #__VA_ARGS__)
+
+int main() {} // COMPILE-ONLY

```

Comment:
This test should be named `test.compile.pass.cpp` so the GitHub harness doesn't bother trying to run it.